modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx

```diff
@@ -57,6 +57,9 @@ const CaseResult = () => {
 		[caseResult.attachments]
 	);
 
+	const buttonMustBeVisible =
+		Object.keys(caseResult.actions).length >= 3 ?? false;
+
```

Comment:
You just need to look for `edit` permission not the rest

```suggestion
	const hasCaseResultEditPermission =
		!!caseResult.actions?.edit
```

Reply:
there is no edit permission, but it contains updateBatch is that right?